packages/sit-onyx/src/components/OnyxBreadcrumbItem/types.ts

```diff
@@ -16,4 +13,4 @@ export type OnyxBreadcrumbItemProps = DensityProp & {
    * Whether to show a skeleton item.
    */
   skeleton?: SkeletonInjected;
-};
+} & Partial<Pick<SharedLinkProps, "href">>;
```

Comment:
```suggestion
} & WithLinkProp;
```

(optional) we could also support changing the target of the link

Reply:
Would be a breaking change then to rename the prop from href to link